omero_search_engine/validation/results_validator.py

```diff
@@ -40,6 +40,7 @@
     "project_name": query_images_in_project_name,
     "screen_name": query_images_screen_name,
     "query_image_or": query_image_or,
+    "in_clause": query_image_or,
```

Comment:
Do we need to rename the template validator in that case to something more generic?

Reply:
I am not sure, what do you suggest?

Reply:
I have fixed the typos and pushed it.

Reply:
Since the  sql call use ``in`` clause. maybe ``query_image_in`` for both.

Reply:
Agree, I have renamed it to `query_image_in `